pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -59,35 +53,7 @@ func CreateK8sBindingInjector() (BindingInjector, error) {
 }
 
 func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *brokerapi.Credential) error {
-	is := makeInjectionSet(binding, cred)
-
-	if err := b.injectSecret(is.secret); err != nil {
-		return err
-	}
-
-	return nil
-}
-
-func (b *k8sBindingInjector) Uninject(binding *servicecatalog.Binding) error {
-	return fmt.Errorf("Not implemented")
-}
-
-func (b *k8sBindingInjector) injectSecret(s *v1.Secret) error {
-	cmc := b.client.Core().Secrets("default")
-	_, err := cmc.Create(s)
-	return err
-}
-
-func makeInjectionSet(binding *servicecatalog.Binding, cred *brokerapi.Credential) *injectionSet {
-	secret := makeSecret(binding, cred)
-
-	return &injectionSet{
-		secret: secret,
-	}
-}
-
-func makeSecret(binding *servicecatalog.Binding, cred *brokerapi.Credential) *v1.Secret {
-	return &v1.Secret{
+	secret := &v1.Secret{
 		ObjectMeta: v1.ObjectMeta{
 			Name: binding.Name,
```

Comment:
ok, I think I finally understand why this simple thing was throwing me. The use of the word "Ref" in there implied something like a pointer (defined by the system) and therefore its something that is set after the secret is created.  What I think you guys are saying is that SecretRef isn't really system defined - its actually user defined but if not specified we'll overwrite it with the binding.Name.

If that's right... I then have a UX question... is this normally how we ask users to provide us the name of the resource they want us to create?  For example, I would have normally expected to see binding.SecretName as the field a user would set.  Coming at this fresh, a name like "SecretRef' would imply we're asking the user to give us the reference to an existing object, not the name (or ref) of a yet to be created object. But if this is a normal Kube thing then ok.  Just wondering.